src/NServiceBus.Core/Unicast/Config/LoadMessageHandlersExtensions.cs

```diff
@@ -2,7 +2,6 @@ namespace NServiceBus
 {
```

Comment:
shouldn't these extensions be updated to make use of the config class?

Reply:
As long as the signature remains the same, why not.
Do you want to use `ReceiveComponent.Configuration.ExecuteTheseHandlersFirstSettingKey` instead of the hardcoded connection string or a different approach?

Reply:
Changed